pkg/resources/view_acceptance_test.go

```diff
@@ -923,6 +923,63 @@ func TestAcc_View_Issue3073(t *testing.T) {
 	})
 }
 
+func TestAcc_View_IncorrectColumnsWithOrReplace(t *testing.T) {
```

Comment:
nit: can you please reference the GH issue here, it helps to give context when someone will be studying these tests later not from the PR context

Reply:
I'll do this in the next PR.